lambdas-py/tests/integration/test_steam_client_integration.py

```
@@ -0,0 +1,286 @@
+    first_game = games[0]
+    assert isinstance(first_game, SteamOwnedGame)
+    assert isinstance(first_game.appid, int)
+    assert first_game.appid > 0, "App ID should be positive"
+    assert first_game.name is not None, "Game name should be present"
+    assert len(first_game.name) > 0, "Game name should not be empty"
+    assert isinstance(first_game.playtime_forever, int)
+    assert first_game.playtime_forever >= 0, "Playtime cannot be negative"
+
+    # Validate at least some games have played time (for realistic test)
+    games_with_playtime = [g for g in games if g.playtime_forever > 0]
+    print(f"\n📊 Library Stats:")
```

Ruff (0.14.8)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)